CHANGES.txt

```diff
@@ -27,6 +27,10 @@ Breaking Changes
 Changes
 =======
 
+ - Introduce support for single row-single column subselects in ANY, e.g.::
```

Comment:
can't we just write: "Added support for subselects as an argument to the ANY function"

bg: it is not true that we only support single row, and multicol will not be implemented i guess.

i would also probably leave out the example, which is btw multirow :-)